src/commands/new.ts

```diff
@@ -89,7 +89,7 @@ const createBlimpProject = async (toolbox: Toolbox, opts: Options) => {
 
   // Create React Native project
   const createSpinner = print.spin('Creating your React Native Project...');
-  await system.run('git clone git@github.com:teamairship/airfoil-template-blimp.git');
+  await system.run('git clone https://github.com/teamairship/airfoil-template-blimp.git');
```

Comment:
I'm curious - what's the reasoning behind changing to https?

Reply:
Changing it to `https` allows anyone to clone the repo without getting an authentication error